scripts/format_code.py

```diff
@@ -175,14 +173,31 @@ def validate_arguments():
       print('ERROR:  -git_diff not defined, and there are no file or')
       print('directory search targets.')
       print('Nothing to do. Exiting.')
-      print()
       return False
   return True
 
+def is_file_objc_header(filename):
+  """Checks the contents of the file to determine if it contains language 
+  constructs that appear in obj-c but not in C/C++.
+  
+  Args:
+    filename (string): the name of the file to check
+  
+  Returns:
+    bool: True if the header file contains known obj-c language definitions.
+    Returns False otherwise.
+  """
+  if '.h' not in filename:
```

Comment:
nit: typically it's best to use one quote style throughout the script. ' vs "

Reply:
Noted.

Reply:
nit: a safer and easier to read check would be to exactly compare the extension
```
_, ext = os.path.splitext(filename)
if ext != '.h':
   return False
```

Reply:
Done